modules/workspace-impl/src/main/java/org/opencastproject/workspace/impl/WorkspaceImpl.java

```diff
@@ -110,6 +114,12 @@
   /** Unknown file name string */
   private static final String UNKNOWN_FILENAME = "unknown";
 
+  /** Key defining the asset manager root directory */
+  private static final String CONFIG_ASSET_MANAGER_ROOT = "org.opencastproject.episode.rootdir";
```

Comment:
Having implemented a similar thing for DCE back in the day, we really, really need to centralize these keys into common, or somewhere.